nonCisMaleInCs/package.json

```diff
@@ -11,7 +11,9 @@
   },
   "dependencies": {
     "react": "16.13.1",
-    "react-native": "0.63.3"
+    "react-native": "0.63.3",
+    "react-native-navigation": "^7.4.0",
+    "typescript": "^4.1.2"
```

Comment:
do we need `typescript` ?

Reply:
Yeah, it was required as a dependency for installing `react-native-navigation`